client/src/components/Playlist/Playlist.jsx

```diff
@@ -39,15 +41,14 @@ export default function Playlist({sessionToken}) {
         setBookResults([])
       }
 
-      async function getRecs(e) {
+      async function getRecs(e, b) {
```

Comment:
these parameter names don't tell me anything about what they represent, please take care in using well thought out parameter names.